mkdocs.yml

```diff
@@ -1,16 +1,49 @@
+# Build settings
+
+# Halt processing when a warning is raised when building the docs.
+# https://www.mkdocs.org/user-guide/configuration/#strict
+strict: true
+
+# Set the remote branch to commit to when using `gh-deploy` to deploy to GitHub Pages.
+# https://www.mkdocs.org/user-guide/configuration/#remote_branch
+remote_branch: gh-pages
+
+# General site info
+
+# Metadata
+
+# Set the main title for the project.
+# Required setting.
+# https://www.mkdocs.org/user-guide/configuration/#site_name
 site_name: Renovate Docs
+
+# Set the canonical URL of the site.
+# https://www.mkdocs.org/user-guide/configuration/#site_url
 site_url: 'https://docs.renovatebot.com'
-site_description: 'Renovate documentation.'
 
-remote_branch: gh-pages
+# Set the site description, this will add a meta tag to the generated HTML header.
+# https://www.mkdocs.org/user-guide/configuration/#site_description
+site_description: 'Renovate documentation.'
 
-strict: true
+# Upstream Git repository information
+# https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/
+repo_name: renovatebot/renovate
+repo_url: https://github.com/renovatebot/renovate
+edit_uri: edit/main/docs/usage/
```

Comment:
@HonkingGoose Can this be also set in markdown files yaml metadata? so we can excplicit set this for the module details pages?

eg for `ansible` manager:

```md
---
edit_uri: edit/main/lib/manager/ansible/readme.md
---

Supports Docker-type dependency extraction from Ansible configuration files.

If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.

...

```

Reply:
OK, seems to not be possible 😕